### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by kambereBr**
In this pull request, the features below are implemented :

1. Add, update and delete items to TodoList
2. Storing data in local storage
3. Use of ES6 modules
4. HTML, CSS & JavaScript  best practices

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+function Task(description, completed, index) {
+  this.description = description;
+  this.completed = completed;
+  this.index = index;
+}
+
+const updateLocalStorage = (data) => {
+  data.forEach((itemTask, index) => {
+    itemTask.index = index + 1;
+  });
```
Comment (Tufahel) [inline src/utils.js:12]: - [ ] As per the project requirements `all changes in the to-do list should be updated in the local storage`.  But I can see when I `check mark` a description it should be changed to `true` in the `localstorage` but it is still `false`.

- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @kambereBr :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Professional readme.
✔️ Add task implemented.
✔️ Delete task implemented.
✔️ Edit task implemented.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/utils.js (hunk 1 above)
Comment (kambereBr) [reply src/utils.js:12]: Thanks @Tufahel for your review

**Review by DuaneDave — APPROVED** [review]
Hi @kambereBr 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ The state of your todo are correctly stored

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**